### PR TITLE
docs(useFetch): minor punctuation fixes

### PR DESCRIPTION
**Pull request by timursaurus**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/3.api/1.composables/use-fetch.md [hunk 1]
```diff
@@ -36,21 +36,21 @@ type AsyncData<DataT> = {
 
 ## Params
 
-* **Url**: The URL to fetch
+* **Url**: The URL to fetch.
 * **Options (extends [unjs/ohmyfetch](https://github.com/unjs/ohmyfetch) options & [AsyncDataOptions](/api/composables/use-async-data#params))**:
-  * `method`: Request method
-  * `params`: Query params
+  * `method`: Request method.
+  * `params`: Query params.
```
Comment (danielroe) [inline docs/content/3.api/1.composables/use-fetch.md:42]: I'm not sure these should have full stops.

File: docs/content/3.api/1.composables/use-fetch.md [hunk 2]
```diff
@@ -11,12 +11,12 @@ function useFetch(
...
-  key?: string,
-  method?: string,
-  params?: SearchParams,
+  key?: string
+  method?: string
+  params?: SearchParams
   body?: RequestInit['body'] | Record<string, any>
-  headers?: {key: string, value: string}[],
-  baseURL?: string,
+  headers?: {key: string, value: string}[]
```
Comment (danielroe) [inline docs/content/3.api/1.composables/use-fetch.md:18]:
```suggestion
  headers?: { key: string, value: string }[]
```

Comment (timursaurus) [reply docs/content/3.api/1.composables/use-fetch.md]: I stumbled upon this too. I checked other files and there were no spaces, I left it untouched

**Review by pi0 — APPROVED** [review]
Thanks!

File: docs/content/3.api/1.composables/use-fetch.md (hunk 2 above)
Comment (danielroe) [reply docs/content/3.api/1.composables/use-fetch.md]: Hm. Not sure what @pi0 thinks, but I'd welcome a pr fixing those other files.

Comment (pi0) [reply docs/content/3.api/1.composables/use-fetch.md]: Yeah generally makes sense to be consistant with code formatting between source and docs. Feel free to open a new PR @timursaurus for this <3

**Pull request merged**